test/holder.go

```diff
@@ -124,3 +124,59 @@ func (h *Holder) MustCreateRankedFragmentIfNotExists(index, field, view string,
 	}
 	return &Fragment{Fragment: frag}
 }
+
+// Row returns a Row for a given field.
+func (h *Holder) Row(index, field string, slice, rowID uint64) *pilosa.Row {
+	idx := h.MustCreateIndexIfNotExists(index, pilosa.IndexOptions{})
+	f, err := idx.CreateFieldIfNotExists(field, pilosa.FieldOptions{})
+	if err != nil {
+		panic(err)
```

Comment:
I think I'm OK with it, but how did you decide to panic here vs passing `t *testing.T` and doing a t.Errorf or t.Fatalf

Reply:
No specific reason. I just followed the same pattern as the other methods in `test/holder.go`.